library/std/src/os/windows/process.rs

```diff
@@ -289,10 +293,11 @@ pub trait CommandExt: Sealed {
     ///
     /// Remember that improper use of raw attributes can lead to undefined behavior or security vulnerabilities. Always consult the documentation and ensure proper attribute values are used.
     #[unstable(feature = "windows_process_extensions_raw_attribute", issue = "114854")]
-    unsafe fn raw_attribute<T: Copy + Send + Sync + 'static>(
+    unsafe fn raw_attribute<T: 'static>(
         &mut self,
         attribute: usize,
-        value: T,
+        value_ptr: *const T,
+        value_size: usize,
     ) -> &mut process::Command;
```

Comment:
I guess the current thinking is to stabilize `raw_attribute` without any of the reserved parameters of `UpdateProcThreadAttribute` exposed? `dwFlags`, `lpPreviousValue`, `lpReturnSize`. That seems fine to me, but can you predict how we'd react if those started being used by Windows? A `CommandExt::raw_attribute_with_flags`, for example? Or a callback kind of thing like unix's `CommandExt::pre_exec`, but where it gives you access to the `lpAttributeList` so that you can "do it yourself" with any arbitrary call to `UpdateProcThreadAttribute`?